misk-core/src/main/kotlin/misk/config/Config.kt

```diff
@@ -8,5 +8,5 @@ package misk.config
 //    "wisp.config.Config"
 //  )
 // )
-@Deprecated("Use from misk-config instead")
+@Deprecated("Use from wisp-config instead")
```

Comment:
why don't we use the deprecation message that's commented?

Reply:
I don't know. It was added a long time ago.